src/main/java/com/sourcegraph/cody/SubscriptionPanel.kt

```diff
@@ -2,10 +2,33 @@ package com.sourcegraph.cody
 
 import com.intellij.ide.BrowserUtil
 import com.intellij.ide.ui.laf.darcula.ui.DarculaButtonUI
+import com.intellij.ui.JBColor
 import com.intellij.ui.dsl.builder.panel
+import com.sourcegraph.common.UpgradeToCodyProNotification
 import com.sourcegraph.config.ConfigUtil
 
 fun createSubscriptionTab(isCurrentUserPro: Boolean) = panel {
+  val chatLimitError = UpgradeToCodyProNotification.chatRateLimitError.get()
+  val autocompleteLimitError = UpgradeToCodyProNotification.autocompleteRateLimitError.get()
+  if (chatLimitError != null && autocompleteLimitError != null) {
+    row {
+      label(
+          "<html>\u26A1 You used all you autocompletions and chats for this month. Upgrade to Cody Pro to get unlimited interactions.</html>")
```

Comment:
we must distinguish pro vs free - it is silly to ask someone who already has pro to upgrade 😅

Reply:
For a pro user you may want to use the copy mentioned there: https://github.com/sourcegraph/jetbrains/issues/194.

Reply:
Agree. And since Mikolaj already added those messages in the text bundle please use it instead of pasting the text directly.

Reply:
On the second look, I do not think the same message will fit into subscription tab. 
Still, we should put those messages in the bundle.

Reply:
And last one: is it intentional that in the first case we do not provide exact info about the limits but we do if only one of the limits is reached?

Reply:
Those labels are implemented according to figma design https://www.figma.com/file/m9Yvgv6W3aPRLfXnxgwnEz/JetBrains?type=design&node-id=215-2737&mode=design&t=ZUILhBlencqhC8Fk-0